.eslintrc.json

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": [
+    "next/core-web-vitals",
+    "@strv/react",
+    "@strv/typescript",
+    "@strv/typescript/react"
+  ],
+
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  }
```

Comment:
Some of the TypeScript ESLint rules we add requires that we inform where is our `.tsconfig.json` located at.